modules/persistent-storage-csi-cinder-storage-class.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-cinder.adoc
+
+[id="persistent-storage-csi-cinder_{context}"]
+= Making OpenStack Cinder CSI the default storage class
+
+In {product-title}, the default storage class references the in-tree Cinder driver. The storage class will default to referencing OpenStack Cinder CSI in a subsequent update of {product-title}. Volumes provisioned using the existing in-tree storage class are planned for migration to the OpenStack Cinder CSI storage class at that time.
+
+The OpenStack Cinder CSI driver uses the `cinder.csi.openstack.org` parameter key to support dynamic provisioning.
+
+.Procedure
+To enable the use of OpenStack Cinder CSI in {product-title} {product-version}, you must overwrite the default in-tree storage class by applying the `standard-csi` storage class.
```

Comment:
A little comment for this sentence, it looks like this is the only way to use Cinder CSI.
Actually, we support using Cinder CSI by following two ways:
1. Creating pvc with explicitly specifying storageclass to "standard-csi"
2. Overwrite the default storageclass as we have following steps.
Just suggest to add case1 somehow beside listing the case 2 already. And we recommend case 2.

Reply:
Gotcha...I reworded slightly so that they know of both options, while recommending the one we then document in the following steps.